Habitica/src/main/java/com/habitrpg/android/habitica/ui/fragments/preferences/PreferencesFragment.java

```diff
@@ -155,27 +155,36 @@ private void scheduleNotifications() {
         Calendar cal = Calendar.getInstance();
         cal.set(Calendar.HOUR_OF_DAY, hour);
         cal.set(Calendar.MINUTE, minute);
-        long trigger_time = cal.getTimeInMillis();
 
-        Intent notificationIntent = new Intent(context, NotificationPublisher.class);
-        notificationIntent.putExtra(NotificationPublisher.NOTIFICATION_ID, 1);
```

Comment:
Good to know - although I thought this would be preferable because it avoids repeating code.